README.md

```diff
@@ -1,3 +1,37 @@
 # vmbuild
 
 Utilities for building IncludeOS VMs
+
+### Configure Settings
+
+```
+  conan config install https://github.com/includeos/conan_config.git
+```
+
+##### View remote and profiles installed
+
+* Remotes:
+```
+  conan remote list
+```
+You should have these:
+
+```
+  includeos: https://api.bintray.com/conan/includeos/includeos [Verify SSL: True]
```

Comment:
These arent really needed as your "default" profile is able to build a working vmbuild for you

Reply:
Removed the texts on remotes.